doc/src/docs/InputOutputReference/01b-InputOutputReference.md

```diff
@@ -17369,16 +17360,13 @@ WaterHeater:HeatPump,
     ,                        !- Inlet Air Temperature Schedule Name
     ,                        !- Inlet Air Humidity Schedule Name
     ,                        !- Inlet Air Zone Name
-    WaterHeater:Mixed,       !- Tank Object Type
     HPWHPlantTank,           !- Tank Name
```

Comment:
Same old question of requiring developers to provide type and name for objects. That requirement was probably never added to the developers guide since in a case like this the object type has only 1 choice.